build.gradle

```diff
@@ -242,7 +242,7 @@ dependencies {
     compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.reformS2sClient
     compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: versions.reformHealthStarter
 
-    compile group: 'uk.gov.hmcts.reform', name: 'data-ingestion-lib', version: '0.3.6'
+    compile group: 'uk.gov.hmcts.reform', name: 'data-ingestion-lib', version: '0.3.9-beta'
```

Comment:
will need to revert this once testing is done?

Reply:
yes